src/Sulu/Bundle/AdminBundle/Resources/js/components/ImageSectorSelection/ImageSectorSelection.js

```diff
@@ -0,0 +1,130 @@
+// @flow
+import {action, computed, observable} from 'mobx';
+import React from 'react';
+import RectangleSelection from '../RectangleSelection';
+import RoundingNormalizer from '../RectangleSelection/dataNormalizers/RoundingNormalizer';
+import type {SelectionData} from '../RectangleSelection/types';
+import log from 'loglevel';
+import {observer} from 'mobx-react';
+import selectionStyles from './imageSectorSelection.scss';
+
+@observer
+export default class ImageSectorSelection extends React.PureComponent {
+    props: {
+        /** Determines the position at which the selection box is rendered at the beginning. */
+        initialSelection?: SelectionData,
+        minWidth?: number,
+        minHeight?: number,
+        onChange?: (s: SelectionData) => void,
+        src: string,
+    };
+
+    image: Image;
+    rounding = new RoundingNormalizer();
+
+    @observable imageLoaded = false;
+    @observable containerWidth: number;
+    @observable containerHeight: number;
+
+    naturalHorizontalToReal = (h: number) => h * this.imageResizedWidth / this.image.naturalWidth;
+    realHorizontalToNatural = (h: number) => h * this.image.naturalWidth / this.imageResizedWidth;
+    naturalVerticalToReal = (v: number) => v * this.imageResizedHeight / this.image.naturalHeight;
+    realVerticalToNatural = (v: number) => v * this.image.naturalHeight / this.imageResizedHeight;
+
+    naturalDataToReal(data: SelectionData): SelectionData {
+        return {
+            width: this.naturalHorizontalToReal(data.width),
+            height: this.naturalVerticalToReal(data.height),
+            left: this.naturalHorizontalToReal(data.left),
+            top: this.naturalVerticalToReal(data.top),
+        };
+    }
+
+    realDataToNatural(data: SelectionData): SelectionData {
+        return {
+            width: this.realHorizontalToNatural(data.width),
+            height: this.realVerticalToNatural(data.height),
+            left: this.realHorizontalToNatural(data.left),
+            top: this.realVerticalToNatural(data.top),
+        };
+    }
+
+    @action componentWillMount() {
+        this.image = new Image();
+        this.image.onload = action(() => this.imageLoaded = true);
+        this.image.onerror = () => log.error('Failed to preload image ' + this.props.src);
+        this.image.src = this.props.src;
+    }
+
+    @computed get imageResizedHeight(): number {
```

Comment:
The next two methods are used to determine the correct scaled-down size of the image. This could probably be done with setting the height or the width of the img to 100 %, depending on whether the image touches the horizontal or vertical borders. Moreover, the case of the image being smaller than the container has to be covered specially, because in that case neither the height nor the width can be set to 100 % without distorting the image. So the logic would be the same, only the return value would be different, when deciding to use percentages instead of pixels.